tests/conftest.py

```diff
@@ -64,6 +71,23 @@ def setup_test(test_dir: Path, macaron_path: Path) -> NoReturn:  # type: ignore
     defaults.clear()
 
 
+@pytest.fixture()
+def check_result() -> CheckResult:  # pylint: disable=unused-argument
+    """Create a CheckResult instance.
+
+    Parameters
+    ----------
+    setup_test
```

Comment:
The doc is inconsistent with the parameter provided to this fixture.

Reply:
I think this fixture should be passed the `setup_test` parameter.

Reply:
Fixed in [29cdd3a](https://github.com/oracle-samples/macaron/pull/67/commits/29cdd3a9e361a0f5f29c835805a2d9f186b92d0a).